spec/fabricators/workshop_fabricator.rb

```diff
@@ -7,7 +7,7 @@
               workshop: workshop,
               sponsor: Fabricate(:sponsor,
                                  seats: transients[:student_count] || 10,
-                                 number_of_coaches: transients[:coach_count || 10]),
+                                 number_of_coaches: transients[:coach_count] || 10),
```

Comment:
@joaoguiIherme do we need the closing parenthesis at the end of this line? I think the next line (11) closes the parenthesis opened on line 8.